Backend/CourseService.cs

```diff
@@ -49,7 +49,8 @@ public async Task<Course> FindByCodeAsync(string code)
 
         public async Task<Course> FindByNameAsync(string name)
         {
-            return await _courseCollection.Find(x => x.CourseName.Equals(name.ToLower())).FirstOrDefaultAsync();
+
+            return await _courseCollection.Find(x => x.CourseName.Equals(name)).FirstOrDefaultAsync();
```

Comment:
It is not necessary already to search case insensitive? (or it is storred everything uppercase?)

Reply:
Course names are stored as we got them, they are not converted into lower case, converting to lower case cause issues when some courses were searched during testing. I also check all usages of this code and found code where we add missing courses, they are added without transforming into lower cases, but most of courses are in lower case, only few have part that are not lower case.

Reply:
The question is, if our GUI will search for course case insensitive. It can be done, a way that we store all codes lower (or upper) or make it lower during comparisons. 
If this code has nothing to do with case insensitive search than forget it...

Reply:
This has nothing to do with search users can do case insensitive during searching for subject. This function "FindByNameAsync" is used only in testing part during start up where database is tested if there are some subjects and then when new course is discovered, when new code is discovered, it was searching for name of subject but it tranformed it to lower case, if subject was not found in lower case, it added that exact subject name but without transforming it to lower case. So it would chec for subject in lower case but it would save it in regular state. I just wanna say that it was searching in lower case but we store subjects without lower case, there are only few iregulation with upper case shortcuts in names lite "technicke prostriedky PC"

Reply:
So there will not exist two subjects in the database with same name (difference just in lower case)?
Because you have removed `name.ToLower()` from the code...

Reply:
There should not be two with same name. We want to exist two or more subject with same name only if they are different in course code or something else that would suggest it is two different courses right?  But i just do not understant why there was name.ToLower() for search when we are storing them in original form.

Reply:
Do can student add some subject we will not found on uniza?

Reply:
No he can not. Studen himself can add only already existing subjects.